Required prerequisites to install dbt core and fusion for BigQuery (#8444)
fishtown-analytics/docs.getdbt.com: website/docs/docs/core/connect-data-platform/bigquery-setup.md

<SetUpPages meta={frontMatter.meta} />

> ## Required Prerequisites

Reviewer: ```suggestion
## Prerequisites
```

> - [Git](https://git-scm.com/install/)

Reviewer: ```suggestion
- Install [Git](https://git-scm.com/install/)
```

> - [Google Cloud SDK](https://docs.cloud.google.com/sdk/docs/install-sdk)

Reviewer: ```suggestion
- Install [Google Cloud SDK](https://docs.cloud.google.com/sdk/docs/install-sdk)
```

> - Python <=3.13.x (Python 3.14 is not supported as yet)

Reviewer: ```suggestion
- Use Python versions 3.13.x or older. Python 3.14 is [not supported](/faqs/Core/install-python-compatibility#python-compatibility-matrix) as yet
```

> - Access to a GCP Account BigQuery Project

Reviewer: ```suggestion
- Access to a GCP account BigQuery project
```

> - Access to a Git Platform (GitHub, AzureDevOps, GitLab, etc) repository.

Reviewer: ```suggestion
- Access to a Git platform repository (like GitHub, AzureDevOps, GitLab, and so on).
```

> - Access to a Git platform repository (like GitHub, AzureDevOps, GitLab, and so on).

Reviewer: ```suggestion
- Access to a Git platform repository (like GitHub, AzureDevOps, GitLab, and so on)
```
